daemon/src/model/cfd.rs

```diff
@@ -982,11 +1089,22 @@ impl Cfd {
                 // TODO: Deal with failed contract setup
                 self.during_contract_setup = false;
             }
+            RolloverStarted => {
+                self.during_rollover = true;
+            }
+            RolloverAccepted => {} /* TODO: It's weird we have no state change for applying an */
+            // event...
             RolloverCompleted { dlc } => {
                 self.dlc = Some(dlc);
+                self.during_rollover = false;
+            }
+            RolloverFailed { .. } => {
+                // TODO: Deal with failed rollover
```

Comment:
@da-kami: Also removing this TODO. What are we not doing that we should be doing? Then I can open an issue.